server/src/test/java/org/elasticsearch/cluster/routing/allocation/ThrottlingAllocationTests.java

```diff
@@ -299,6 +301,15 @@ public void testOutgoingThrottlesAllocation() {
             new MoveAllocationCommand("test", 0, "node2", "node4")), true, false);
         assertEquals(commandsResult.explanations().explanations().size(), 1);
         assertEquals(commandsResult.explanations().explanations().get(0).decisions().type(), Decision.Type.THROTTLE);
+        boolean beingOutgoingThrottled = false;
+        for (Decision decision : commandsResult.explanations().explanations().get(0).decisions().getDecisions()) {
+            if (decision.label().equals(ThrottlingAllocationDecider.NAME)) {
+                assertThat(decision.getExplanation(), containsString("node [node1] which holds the primary"));
```

Comment:
Could you assert the entire text of the message as you had done previously? It helps to clarify what this bit of code is testing.